version-figures.sh

```diff
@@ -10,6 +10,7 @@ echo "Using figure versions from commit $FIGURE_COMMIT_SHA"
 CSSE_STATS_JSON=csse/csse-stats.json
 EBM_STATS_JSON=ebmdatalab/ebmdatalab-stats.json
 CORD19_STATS_JSON=CORD-19/cord19-stats.json
+CONTRIB_STATS_JSON=contrib-viz/covid19-review-stats.json
```

Comment:
I don't think this file needs to be modified yet.  We don't have the figure URLs in `covid19-review-stats.json` so they don't need to have the commit replaced in the URLs.  We could save that set of changes for another pull request that integrates this into the daily or weekly workflow.

Reply:
OK that sounds great!